app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if not parts or parts[0] != "mv" or len(parts) != 3:
+        return
```

Comment:
it is better to raise some exception so that it is clear to the user if something goes wrong